medusa/scene_numbering.py

```diff
@@ -495,17 +493,18 @@ def xem_refresh(indexer_id, indexer, force=False):
             if not indexerApi(indexer).config.get('xem_origin'):
                 return
             # XEM MAP URL
-            url = "http://thexem.de/map/havemap?origin=%s" % indexerApi(indexer).config['xem_origin']
+            url = "http://thexem.de/map/havemap?origin={0}".format(indexerApi(indexer).config['xem_origin'])
             parsedJSON = helpers.getURL(url, session=xem_session, returns='json')
             if not parsedJSON or 'result' not in parsedJSON or 'success' not in parsedJSON['result'] or 'data' not in parsedJSON or str(indexer_id) not in parsedJSON['data']:
+                logger.log(u'No XEM data for show "{0} on {1}"'.format(indexer_id, indexerApi(indexer).name), logger.INFO)
```

Comment:
I added this one @p0psicles becouse it was showing when that it was contacting xem but none that there were no results. The other lines I only changed to use .format()